oak/proto/storage_channel.proto

```diff
@@ -16,73 +16,9 @@
 
```

Comment:
perhaps this file should be renamed to `storage_node` or `storage_service` at some point? (Or just deleted -- hint hint)

Reply:
Seems reasonable, but for another day.

Reply:
Actually, done: it's now `StorageService` in `oak/proto/storage_service.proto`